sample_information_validator.py

```diff
@@ -51,44 +60,97 @@ def validate(self, cell):
             return False
         except TypeError:
             if cell.value is None:
-                logger.warning(
+                logger.error(
                     'Cell {} is numeric but empty'.format(cell.coordinate)
                 )
+                return False
             else:
                 raise
 
+    def validate_vol(self,cell): # TODO different volumes for different input types! requires changes in data sheet
```

Comment:
We could have the input type as a argument given to the script on the command line. At least that could be a start and then it would be easy to replace that when we have a different source for that information.